Backend/Microservices/AuthService/src/Auth.Application/Interfaces/IUserRepository.cs

```diff
@@ -0,0 +1,19 @@
+﻿using Auth.DAL.Entities;
+using Microsoft.AspNetCore.Identity;
+
+namespace Auth.BLL.Interfaces
+{
+    public interface IUserRepository
+    {
+        Task Create(User user, CancellationToken cancellationToken);
+        Task Delete(User user, CancellationToken cancellationToken);
+        Task<List<User>> Get(CancellationToken cancellationToken);
```

Comment:
The Get methods are a bit confusing. Get should be GetAllUsers, and Get(long userId) should be GetById for clarity.